### PR TITLE
build: Dependency updates

### PR DESCRIPTION
**Pull request by romtsn**

### CODE REVIEW

**Review by philipphofmann — APPROVED** [review]
Good job 👏

File: build.gradle.kts [hunk 1]
```diff
@@ -55,20 +44,17 @@ kotlin {
...
-                implementation("androidx.core:core-ktx:1.3.1")
+                implementation("io.sentry:sentry-android:5.0.1")
             }
         }
 
         val jvmMain by getting {
             dependencies {
-                implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version")
-                implementation("io.sentry:sentry-core:2.3.0")
+                implementation("io.sentry:sentry-core:5.0.1")
```
Comment (marandaneto) [inline build.gradle.kts:53]:
```suggestion
                implementation("io.sentry:sentry:5.0.1")
```

-core does not exist anymore

Comment (philipphofmann) [reply build.gradle.kts:53]: ups 🙈

Comment (philipphofmann) [reply build.gradle.kts:53]: Fixed in https://github.com/getsentry/sentry-kotlin-multiplatform/pull/6

**Pull request merged**